### ISSUE TITLE
Installer not recognised by Windows SmartScreen

### ISSUE THREAD

**Issue by abraaoprogramador** [HaxeFoundation/haxe#9073]
Hello! I downloaded the haxe-4.0.5-win64.exe from here: https://haxe.org/download/ and the Windows SmartScreen shows the publisher as "Unknown". It's the same issue as #4211. I commented there but I was uncertain if was necessary to create another issue so I did create.
I'm running it in Windows 10.

> Windows protected your PC
Windows SmartScreen prevented an unrecognised application from starting. Running this application > might put your PC at risk."

>

**Comment by RealyUniqueName** [comment]
I don't know how to workaround SmartScreen. 
From what I've seen in google I concluded that it requires to sign our installers with a certificate. We have an issue for that: #7720

**Comment by RealyUniqueName** [comment]
I'm closing this since there's no way to "fix" it.
Any new software has to "gain reputation" to not get blocked by SmartScreen.
Signing an app may help, and as I said we already have an issue for that.
https://www.digicert.com/blog/ms-smartscreen-application-reputation/
